docs/source/index.rst

```diff
@@ -114,7 +110,10 @@ How to cite
 
 If you use AiiDA for your research, please cite the following work:
 
-.. highlights:: Giovanni Pizzi, Andrea Cepellotti, Riccardo Sabatini, Nicola Marzari,
+.. highlights:: **AiiDA 1.0:** Sebastiaan. P. Huber, Spyros Zoupanos, Martin Uhrin, Leopold Talirz, Leonid Kahle, Rico Häuselmann, Dominik Gresch, Tiziano Müller, Aliaksandr V. Yakutovich, Casper W. Andersen, Francisco F. Ramirez, Carl S. Adorf, Fernando Gargiulo, Snehal Kumbhar, Elsa Passaro, Conrad Johnston, Andrius Merkys, Andrea Cepellotti, Nicolas Mounet, Nicola Marzari, Boris Kozinsky, Giovanni Pizzi, *AiiDA 1.0, a scalable computational infrastructure for automated reproducible workflows and data provenance*, `arXiv:2003.12476 (2020) <https://arxiv.org/abs/2003.12476>`_;
```

Comment:
Is the intention that you should cite this paper when using AiiDA 1.0+ and the other one for < 1.0? If yes, I don't think that's clear enough. If that's not the intention, it's perfectly fine.

Reply:
Probably yes, but I think it's ok to merge this for now, also considering that the paper is not really published; we will have anyway to fix this when the paper is out, and also there will be an AiiDA engine paper out soon and we need to clarify this further on what to cite when.